examples/app-router/next.config.js

```diff
@@ -51,6 +51,13 @@ const nextConfig = {
         permanent: true,
         has: [{ type: "cookie", key: "from", value: "wrongvalue" }],
       },
+      {
+        source: "/redirect",
```

Comment:
`/redirect` is already used in the middleware for a redirect. This makes the middleware redirect test fail